golang/cosmos/app/app.go

```diff
@@ -929,9 +941,12 @@ type cosmosInitAction struct {
 	vm.ActionHeader `actionType:"AG_COSMOS_INIT"`
 	ChainID         string          `json:"chainID"`
 	IsBootstrap     bool            `json:"isBootstrap"`
-	UpgradeDetails  *upgradeDetails `json:"upgradeDetails,omitempty"`
 	Params          swingset.Params `json:"params"`
-	SupplyCoins     sdk.Coins       `json:"supplyCoins"`
+	// ResolvedConfig is the subset of complete config that is relevant to the VM.
+	// It is a superset of swingset.SwingsetConfig.
```

Comment:
How is it a superset?

Reply:
Right now it's a superset only in the trivial sense, but we know that cosmic-swingset _will_ need data that's not under `[swingset]` (at which point it will be a strict superset), so I just documented that now.

Reply:
I'm curious as to what "config" options would be purely generated instead of resolved.

Reply:
I was thinking of global settings like `--moniker`, `--log_level`, and `--log_format`, but I guess even if we did want those they'd be communicated at invocation rather than in the AG_COSMOS_INIT message. I'll simplify.